modules/twinklespeedy.js

```diff
@@ -248,6 +270,10 @@ Twinkle.speedy.callback.modeChanged = function twinklespeedyCallbackModeChanged(
 
 	var radioOrCheckbox = (Twinkle.speedy.mode.isMultiple(mode) ? 'checkbox' : 'radio');
 
+	work_area.append( { type: 'header', label: 'Custom rationale' } );
+
+	work_area.append( { type: radioOrCheckbox, name: 'csd', list: Twinkle.speedy.generateCsdList(Twinkle.speedy.customRational, mode) } );
```

Comment:
Doesn't need to be under a separate header for non-admins. For tagging, simply merge the customRationale array with the generalList array and add them to the general header, to preserve current behaviour. Otherwise you are just making the list longer for no good  reason.